docs/Assets.md

```diff
@@ -140,6 +140,8 @@ Method: GET
 | [options.assetTypeId] | <code>string</code> | UUID of the asset type to use for filtering |
 | [options.limit] | <code>Number</code> | Maximum number of records to return per query |
 | [options.offset] | <code>Number</code> | How many records from the first record to start |
+| [options.includeTypeDescendents] | <code>boolean</code> | When true will look for all asset types that are descendendents from options.assetTypeId |
+| [options.includeMetricsId] | <code>boolean</code> | If asset has an associated metric value the latest metric will be returned. |
```

Comment:
All of these are supported together...I think there are newer options to filter by asset attributes and values.